backend/recruitmentapp/apps/api/v1/serializers.py

```diff
@@ -31,6 +53,8 @@ class ApplicantSerializer(serializers.Serializer):
     )
     email = serializers.EmailField(source='user.email')
     social_security_number = serializers.CharField(max_length=20)
+    competences = CompetenceSerializer(many=True)
+    availability = AvailabilitySerializer(many=True)
```

Comment:
This field should be named `availabilities`, because that is the name of the field on the model.
https://github.com/IV1201-VT18-G8/recruitment-app/blob/dev/backend/recruitmentapp/apps/core/models.py#L53

Alternatively we can change the model, but then we need another migration.